orcoursetrion/config.py

```diff
@@ -32,6 +32,15 @@
 
     # Deployment team for studio
     ('ORC_STUDIO_DEPLOY_TEAM', 'mitx-studio-export'),
+
+    # Org to use for XML/latex2edx courses
+    ('ORC_XML_ORG', 'mitx'),
+
+    # Deployment team for XML/latex2edx
+    ('ORC_XML_DEPLOY_TEAM', 'mitx-content-deployment'),
+
+    # Web hook URL (including basic auth) for course development LMS
+    ('ORC_STAGING_GITRELOAD', None),
 ]
 for key in CONFIG_KEYS:
     value = primary_config.get(key[0], fallback_config.get(key, key[1]))
```

Comment:
Ooops. There is a slight docs issue, that I think @pwilkins brought up. The keys that is looks for aren't named the same as in the README, (but it's right in configuration.)